client/lib/query-manager/paginated/test/index.js

```diff
@@ -269,12 +267,12 @@ describe( 'PaginatedQueryManager', () => {
 			sandbox.stub( PaginatedQueryManager, 'matches' ).returns( false );
```

Comment:
Seems like this still uses `sinon` - care to migrate?

Reply:
👍